### PR TITLE
dev/core/482 standardize contact edit form delete block behavior

### PR DESCRIPTION
**Pull request by elisseck**
Overview
----------------------------------------
When adding a new contact via the "New Individual", "New Organization" etc forms, or editing a contact, the website blocks allow a user to delete a block even if it's the only one left. Afterwards, the user is not able to re-create a website block. This is non-standard for this form i.e. Phone Number, Email, IM do not let a user delete a block if it's the only one left.

Relevant issue: https://lab.civicrm.org/dev/core/issues/482

Before
----------------------------------------
Users can delete the last website block, breaking their ability to add a website.
image

After
----------------------------------------
Users are not given the option to delete the last website block, which is the standard behavior for this form.
image

Technical Details
----------------------------------------
The js to create a new block relies on an existing block of that type to copy. If statement added is present in the templates for phone, IM, email blocks but was missing in the website block template.

### CODE REVIEW

File: templates/CRM/Contact/Form/Edit/Website.tpl [hunk 1]
```diff
@@ -41,7 +41,9 @@
 <tr id="Website_Block_{$blockId}">
     <td>{$form.website.$blockId.url.html|crmAddClass:url}&nbsp;</td>
     <td>{$form.website.$blockId.website_type_id.html}</td>
+    {if $blockId gt 1}
     <td colspan="3"><a href="#" title="{ts}Delete Website Block{/ts}" onClick="removeBlock('Website','{$blockId}'); return false;">{ts}delete{/ts}</a></td>
```
Comment (seamuslee001) [inline templates/CRM/Contact/Form/Edit/Website.tpl]: @elisseck This should probably be indented 2 spaces now its within the if

Comment (elisseck) [reply templates/CRM/Contact/Form/Edit/Website.tpl]: Now I just need to remember how to rebase this properly

**Comment by elisseck** [comment]
While looking at a different, possibly related issue I realized that the website inline form has a different template with the same issue. My second commit should resolve that bit as well.

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by monishdeb** [comment]
This change makes sense, it is as per how we are redendering delete icon for other fields, as in:
```
templates//CRM/Contact/Form/Edit/IM.tpl:44:    {if $blockId gt 1}
<a class="crm-delete-inline crm-hover-button" href="#" title="{ts}Delete IM{/ts}"><span class="icon delete-icon"></span></a>
templates//CRM/Contact/Form/Edit/OpenID.tpl:42:    {if $blockId gt 1}
templates//CRM/Contact/Form/Edit/Phone.tpl:51:  {if $blockId gt 1}
```
Checked in my local works for me : 
screen shot 2018-12-05 at 12 11 57 am

**Pull request merged**